bindings/InterfaceBindings.cpp

```
@@ -60,58 +64,73 @@ struct StatevectorCPP {
+void bindFramework(py::module& m) {
+<<<<<<< HEAD
+  // Bind the Result enum
+  py::native_enum<Result>(m, "Result", "enum.Enum",
+                          "Represents the result of an operation.")
+      .value("OK", OK, "Indicates that the operation was successful.")
+      .value("ERROR", ERROR, "Indicates that an error occurred.")
+      .export_values()
+      .finalize();
+
+=======
+>>>>>>> parent of a462ea5 (♻️ Replace `pybind11` with `nanobind` (#248))
```

Cppcheck (2.19.0)

[error] The code contains unhandled character(s) (character code=226). Neither unicode nor extended ascii is supported.

(unhandledChar)